projects/fluid_simulation/sim.cpp

```diff
@@ -309,6 +310,11 @@ void FluidSimulationApp::SetupRenderingPipeline()
     PPX_CHECKED_CALL(GetDevice()->CreateGraphicsPipeline(&gpci, &mGraphicsPipeline));
 }
 
+void FluidSimulationApp::Resize(uint32_t width, uint32_t height)
+{
+    SetupGrids();
```

Comment:
I assume this destroys and then recreates?

Reply:
Yes.  They're all unique pointers and std containers that `SetupGrids` empties.